tests/test_iotedgedev.py

```diff
@@ -12,24 +13,35 @@
 from iotedgedev.output import Output
 
 from .utility import assert_json_file_equal
+from .utility import get_docker_client
 
 pytestmark = pytest.mark.e2e
 
 root_dir = os.getcwd()
 tests_dir = os.path.join(root_dir, "tests")
-envvars = EnvVars(Output())
+output = Output()
+envvars = EnvVars(output)
 env_file_name = envvars.get_dotenv_file()
 env_file_path = envvars.get_dotenv_path(env_file_name)
 
 test_solution = "test_solution"
 test_solution_dir = os.path.join(tests_dir, test_solution)
 launch_json_file = os.path.join(tests_dir, "assets", "launch.json")
+launch_json_file_without_nodejs = os.path.join(tests_dir, "assets", "launch_without_nodejs.json")
 
 test_solution_shared_lib = "test_solution_shared_lib"
 test_solution_shared_lib_dir = os.path.join(tests_dir, "assets", test_solution_shared_lib)
 
 
-@pytest.fixture(scope="module", autouse=True)
+def get_docker_os_type():
```

Comment:
This method is shared by other tests. You can move it to the utility class.

Reply:
Updated code.